### PR TITLE
LogFile generation in Temp Folder

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**

### CODE REVIEW

**Comment by Samuelopez-ansys** [comment]
Hi @beliaev-maksim ,

Could check why is failing the Unittest? It is working in my machine.

**Comment by beliaev-maksim** [comment]
@Samuelopez-ansys 
did you run `mypy --config-file=./mypy.ini aedttest` on your machine?

File: aedttest/aedt_test_runner.py [hunk 1]
```diff
@@ -914,6 +914,7 @@ def execute_aedt(
 
     """
     aedt_path = get_aedt_executable_path(version)
+    log_path = os.path.splitext(project_path)[0] + ".log"
```
Comment (beliaev-maksim) [inline aedttest/aedt_test_runner.py:917]: basically, 
project_path could be optional according to the type
however, `os.path.splitext` does not support `None`

**Review by beliaev-maksim — CHANGES_REQUESTED** [review]

**Comment by Samuelopez-ansys** [comment]
Thanks @beliaev-maksim, found the error. Just passing -LogFile <""> is creating the log file in the project folder.

File: aedttest/aedt_test_runner.py [hunk 2]
```diff
@@ -914,7 +914,9 @@ def execute_aedt(
 
     """
     aedt_path = get_aedt_executable_path(version)
-    log_path = os.path.splitext(project_path)[0] + ".log"
+    log_path = ""
+    if project_path is not None:
```
Comment (beliaev-maksim) [inline aedttest/aedt_test_runner.py:918]: we already have it below :)

**Comment by beliaev-maksim** [comment]
@Samuelopez-ansys 
great, thanks!

**Pull request merged**